client/rest-high-level/src/main/java/org/elasticsearch/client/xpack/XPackInfoResponse.java

```diff
@@ -88,26 +78,6 @@ public FeatureSetsInfo getFeatureSetsInfo() {
         return featureSetsInfo;
     }
 
-    @Override
-    public boolean equals(Object other) {
-        if (other == null || other.getClass() != getClass()) return false;
-        if (this == other) return true;
-        XPackInfoResponse rhs = (XPackInfoResponse) other;
-        return Objects.equals(buildInfo, rhs.buildInfo)
-                && Objects.equals(licenseInfo, rhs.licenseInfo)
-                && Objects.equals(featureSetsInfo, rhs.featureSetsInfo);
-    }
-
-    @Override
-    public int hashCode() {
-        return Objects.hash(buildInfo, licenseInfo, featureSetsInfo);
-    }
-
-    @Override
-    public String toString() {
```

Comment:
I feel like losing `toString` on these is sad!

Reply:
it relied on `ToXContentObject` in hlrc that is redundant there

Reply:
Not really. It was nice to have because it was free to implement but isn't really worth adding unless we bump into test failures. We can add it then if we need it.

Reply:
ok, reverted it back at  c99b27b